draft-dekater-scion-dataplane.md

```diff
@@ -398,7 +399,7 @@ The SCION common header has the following packet format:
 {: #figure-3 title="The SCION common header packet format"}
 
 - `Version`: The version of the SCION common header. Currently, only version "0" is supported.
-- `TrafficClass` (`TraffCl` in the image above): The 8-bit long identifier of the packet's class or priority. The value of the traffic class bits in a received packet or fragment might differ from the value sent by the packet's source. The current use of the `TrafficClass` field for Differentiated Services and Explicit Congestion Notification is specified in {{RFC2474}} and {{RFC3168}}.
+- `TrafficClass` (`TraffCl` in the image above): The 8-bit long identifier of the packet's class or priority. The value of the traffic class bits in a received packet might differ from the value sent by the packet's source. The current use of the `TrafficClass` field for Differentiated Services and Explicit Congestion Notification is specified in {{RFC2474}} and {{RFC3168}}.
```

Comment:
@jiceatscion Do ypu know why we mention fragment here? I think we should not. 
This text comes from https://github.com/scionproto/scion/blob/74878a477c7821fb66f8d6a39bdddced590bd91d/doc/protocols/scion-header.rst#L35

Reply:
OK, can you also change this in scionproto?

Reply:
Done. see https://github.com/scionproto/scion/pull/4635